configs/components/echam/echam.yaml

```diff
@@ -509,6 +513,26 @@ choose_yr_perp_off:
                                 runctl:
                                         l_orbsvop87: ".false."
 
+choose_wiso_code:
+        True:
+                lwiso: "${with_wiso}"
+                add_namelist_changes:
+                        namelist.echam:
+                                wisoctl:
```

Comment:
does having wiso code compiled in directly require a wisoctl in the namelist? I was not aware of that.

Reply:
Yes, one of the things give users headaches.